src/app/employee/employee-register/employee-register.component.ts

```
@@ -0,0 +1,116 @@
+    return (control: AbstractControl): ValidationErrors | null => {
+      const password = control.get('password');
+      const passwordConfirmation = control.get('passwordConfirmation');
+
+      const password_errors = password?.errors ?? {};
+      const passwordConfirmation_errors = passwordConfirmation?.errors ?? {};
+
+      if (password?.value !== passwordConfirmation?.value) {
+        password_errors['passwordMismatch'] = true;
+        passwordConfirmation_errors['passwordMismatch'] = true;
+      } else {
+        delete password_errors['passwordMismatch'];
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)